ocs_ci/deployment/baremetal.py

```diff
@@ -1251,12 +1251,16 @@ def destroy_cluster(self, log_level="DEBUG"):
 
 
 @retry(exceptions.CommandFailed, tries=10, delay=30, backoff=1)
-def clean_disk(worker, namespace=constants.DEFAULT_NAMESPACE):
+def disks_available_to_cleanup(worker, namespace=constants.DEFAULT_NAMESPACE):
     """
-    Perform disk cleanup
+    disks available for cleanup
 
     Args:
         worker (object): worker node object
+        namespace (str): namespace where the oc_debug command will be executed
+
+    Returns:
+        disk_names_available_for_cleanup (int): No of disks avoid to cleanup on a node
```

Comment:
A small change here: 
`disk_names_available_for_cleanup (list): The disk names available for cleanup on a node`

Reply:
Updated in latest commit.